RELEASE_NOTES.md

```diff
@@ -6,9 +6,29 @@ _This file holds "in progress" release notes for the current release under devel
 
 Chef can now create symlinks without privilege escalation, which allows for the creation of symlinks on Windows 10 Creator Update.
 
-## apt_preference resource
+## nokogiri Gem
 
-The apt_preference resource has been ported from the apt cookbook. This resource allows you to create APT preference files controlling which versions of packages will be installed.
+The nokogiri gem is once again bundled with the omnibus install of Chef
+
+## zypper_package Options
+
+It is now possible to pass additional options to the zypper in the zypper_package resource. This can be used to pass any zypper CLI option
+
+### Example:
+
+```ruby
+zypper_package 'foo'
+  options '--user-provided'
+end
+```
+
+## windows_task Improvements
+
+The `windows_task` resource now properly allows updating the configuration of a scheduled task when using the `:create` action. Additionally the previous `:change` action from the windows cookbook has been aliased to `:create` to provide backwards compatibility.
+
+## apt_preference Resource
+
+The apt_preference resource has been ported from the apt cookbook. This resource allows for the creation of APT preference files controlling which versions of packages will be installed.
 
 Further information regarding apt-pinning is available via <https://wiki.debian.org/AptPreferences>.
```

Comment:
I tweaked it a tiny, but and linked to the man page, which has non pinning information.